packages/app/src/components/routes.tsx

```diff
@@ -39,24 +39,22 @@ export const Routes: React.FC = () => {
   const isSignedIn = !isOnboardingInProgress && identities.length;
 
   const getSignUpElement = () => {
+    const { search } = new URL(location.href);
```

Comment:
Shouldn't this not work? Since the query params are technically part of the hash?